src/com/palmergames/bukkit/towny/command/BaseCommand.java

```diff
@@ -360,4 +360,15 @@ public static void checkPermOrThrow(Permissible permissible, String node) throws
 	public static void checkPermOrThrowWithMessage(Permissible permissible, String node, Translatable errormsg) throws NoPermissionException {
 		TownyUniverse.getInstance().getPermissionSource().testPermissionOrThrow(permissible, node, errormsg);
 	}
+
+	public static List<String> getTrustedTownsFromResidentOrThrow(Player player){

```

Comment:
This method doesn't throw anything so the name isn't correct.

Is this method used anywhere?

Reply:
Name has been changed in latest commit.
This method is used in the add/remove cases for the trusttown tab completer to determine towns which should be filtered for both cases.

Reply:
It might make sense to move it to TownCommand, unless it is also being called from TownyAdminCommand.

Reply:
Agreed, have moved this to the TownCommand class.